usage.py

```diff
@@ -30,6 +30,19 @@
 
     html.Div([
         html.Div([
+            html.Div([
+                html.P([html.Strong('Test for AppBar')]),
+                sd_material_ui.AppBar(
+                    id='appbar',
+                    children=sd_material_ui.Toolbar(
+                        id='toolbar',
+                        children=[
+                            html.H4(children='AppBar'),
```

Comment:
Does `AppBar` always require a `Toolbar` child element? Can `Toolbar` be used in ways other than inside an `AppBar`? If yes, would it be easier for users if the `AppBar` component contained a `Toolbar` automatically, so users don't need to combine them? That might make the separate `Toolbar` component unnecessary, and any props intended for that component could be handled by `AppBar` and passed through to the `Toolbar`.

Reply:
According to the answer to this question: https://stackoverflow.com/questions/52653103/what-is-appbar-vs-toolbar/53749453 AppBar can be used without having a Toolbar child element.

It's just that for my use cases at least I only use the AppBar as a navigation bar with inline 'tools', so that's the example I chose for testing.